context/app/static/js/components/detailPage/ContributorsTable/ContributorsTable.tsx

```diff
@@ -6,32 +6,45 @@ import TableBody from '@mui/material/TableBody';
 import TableCell from '@mui/material/TableCell';
 import TableHead from '@mui/material/TableHead';
 import TableRow from '@mui/material/TableRow';
+import { Check } from '@mui/icons-material';
+import { Stack, Typography } from '@mui/material';
 
 import OutboundIconLink from 'js/shared-styles/Links/iconLinks/OutboundIconLink';
 import { StyledTableContainer, HeaderCell } from 'js/shared-styles/tables';
 import IconTooltipCell from 'js/shared-styles/tables/IconTooltipCell';
 import SectionHeader from 'js/shared-styles/sections/SectionHeader';
 import { DetailPageSection } from 'js/components/detailPage/style';
-import { ContributorAPIResponse } from './utils';
+import EmailIconLink from 'js/shared-styles/Links/iconLinks/EmailIconLink';
+import InfoAlert from 'js/shared-styles/alerts/InfoAlert';
+import { isValidEmail } from 'js/helpers/functions';
+
 import { useNormalizedContributors } from './hooks';
+import { ContributorAPIResponse, sortContributors } from './utils';
 
 interface ContributorsTableProps {
   title: string;
   contributors: ContributorAPIResponse[];
   iconTooltipText?: string;
+  showInfoAlert?: boolean;
 }
 
-function ContributorsTable({ title, contributors = [], iconTooltipText }: ContributorsTableProps) {
+function ContributorsTable({ title, contributors = [], iconTooltipText, showInfoAlert }: ContributorsTableProps) {
   const columns = [
     { id: 'name', label: 'Name' },
     { id: 'affiliation', label: 'Affiliation' },
+    { id: 'contact', label: 'Contact' },
   ];
 
   const normalizedContributors = useNormalizedContributors(contributors);
+  const sortedContributors = sortContributors(normalizedContributors);
+
+  const contributorsInfoAlertText =
```

Comment:
Since this is static text, we can define it outside of the component for a small performance boost (i.e. React won't have to de-allocate and re-allocate the memory for this variable on each render)

Reply:
Got it, good catch!